api_tests/comments/views/test_comment_detail.py

```diff
@@ -139,15 +139,15 @@ def test_comment_has_node_link(self):
         self._set_up_public_project_with_comment()
         res = self.app.get(self.public_url)
         url = res.json['data']['relationships']['node']['links']['related']['href']
-        expected_url = '/{}nodes/{}/'.format(API_BASE, self.public_project._id)
+        expected_url = '/{}registrations/{}/'.format(API_BASE, self.public_project._id)
         assert_equal(res.status_code, 200)
         assert_equal(urlparse(url).path, expected_url)
 
     def test_comment_has_replies_link(self):
         self._set_up_public_project_with_comment()
         res = self.app.get(self.public_url)
-        url = res.json['data']['relationships']['replies']['links']['self']['href']
-        expected_url = '/{}nodes/{}/comments/?filter[target]={}'.format(API_BASE, self.public_project._id, self.public_comment._id)
+        url = res.json['data']['relationships']['replies']['links']['related']['href']
+        expected_url = '/{}registrations/{}/comments/?filter[target]={}'.format(API_BASE, self.public_project._id, self.public_comment._id)
```

Comment:
I don't think this is right. Is public_project a node or a registration? If it's a node, the expect url should go to nodes. If it's a registration, then just be sure that there is an equivalent test for nodes as well.